less/navigation.less

```diff
@@ -1 +1,77 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+
+.border(){
+    border: 1px solid red;
+}
+
+.alignnavitems(){
+    display: flex;
+    align-items: flex-end
+}
+
+
+
+//navcode
+
+.topnav {
+    width: 100%;
+    border-bottom: 2px dashed silver;
+    height: 74px;
+    // .border();
+
+    .navwide{
+        display:flex;
+        max-width: 800px;
+        width: 100%;
+        margin: 0 auto;
+        // .border();
+        justify-content: space-between;
+        .alignnavitems();
+    }
+
+    .links{
+        // .border;
+        .alignnavitems();
+    }
+
+    a {
+        text-decoration: none;
+        text-align: center;
+        color: #212529;
+        font-size: 1.6rem;
+        padding: 20px;
+        padding-bottom:10px;
+    }
+
+    @media @mobile{
```

Comment:
@mobile is a great example of variable that saves us some time not having to rewrite the same code over and over